app/src/main/res/values/strings.xml

```diff
@@ -16,7 +16,10 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="appearance">Appearance</string>
     <string name="rename">Rename</string>
     <string name="confirm_delete">Confirm Delete</string>
-    <string name="do_you_really_want_to_delete_this_witharg">Do you really want to delete this %s?</string>
+    <plurals name="do_you_really_want_to_delete_this_document">
+        <item quantity="one">Do you really want to delete this document?</item>
```

Comment:
please have for both a separate string ID in strings.xml

Reply:
this indivisible string resource should have the single ID

Reply:
I don't want anything but key=value in strings.xml. Other cases have proofed not to work good with translation tools & services.

Reply:
Ok, I can put the resource to other file, but `translatable="false"` shouldn't apply to it. Crowdin supports plurals resources.

Reply:
But others don't. I want to depend neither github, crowdin nor both.